msteams-platform/concepts/build-and-test/share-to-teams-from-web-apps.md

```diff
@@ -21,19 +21,44 @@ The following image displays the pop-up window for Share to Teams preview experi
 
 You can also add link unfurling for the links shared through Share to Teams button hosted in web app, personal app or tab. For more information, see [link unfurling](~/messaging-extensions/how-to/link-unfurling.md).
 
-The following image displays the link unfurling experience through Share to Teams button:
-
-:::image type="content" source="~/assets/images/share-to-teams-link-unfurling.png" alt-text="Share-to-Teams link unfurling":::
-
 This article guides you on how to create and embed a Share to Teams button for your website, craft your website preview, and extend Share to Teams for Education.
 
 See the following video to learn how to embed Share to Teams button:
 <br>
 > [!VIDEO https://www.microsoft.com/en-us/videoplayer/embed/RE4vhWH]
 <br>
 
+## Prerequisites
+
+To share content to meeting stage, you must ensure the following criteria are met:
+
+1. A Teams app with a tab that supports:
+
+    * Meeting side panel
+    * Instant tab
+
+2. To present the content onto the meeting stage, the meeting app must support Share to Stage APIs, which has two major requirements:
+
+    * Support Meeting Stage and Meeting Side Panel within the app manifest.
+    * Support read permissions for taking over the stage `MeetingStage.Write.Chat`.
+
+To share the entire app to stage, you must configure `meetingStage` and `meetingSidePanel` as frame contexts in the [app manifest](../../resources/schema/manifest-schema.md). Otherwise, meeting attendees may not be able to see the content on stage.
+
+<!--## Steps
+
+* Embed Share to Teams button on webpage.
+* In the meeting tab, add additional logic to query shared content (new APIs to be built by Teams platform) and display the content within the tab experience.
+* Allow users to present that content onto the meeting stage via the Share to Stage APIs.
+* Notify users of shared content via link unfurling.-->
+
 ## Embed a Share to Teams button
 
+There are three different methods by which you can embed a Share to Teams button on your web page. Based on the control you want on the Share to Teams button, you can use one of the following methods:
```

Comment:
We will only have two methods. The other `shareInMeetingClickHandler` is specific for share-in-meeting button control. So we will not have that method here.

Reply:
Updated.